### ISSUE TITLE
purpose of elastic search

### ISSUE THREAD

**Issue by HelloTo4x** [Erudika/para-search-elasticsearch#23]
Hi, can u tell me the purpose of elastic search in this project, does it use for Full-text search

**Comment by albogdano** [comment]
Yes, Para uses ES for indexing data and to enable full text search. There is also a plugin for Lucene for the same purpose.

**Comment by albogdano** [comment]
We could, of course. It's just not implemented yet.
